plugins/europeana/record.js

```diff
@@ -12,39 +13,77 @@ function parseRecordDataFromApiResponse(response) {
 
   const providerAggregation = edm.aggregations[0];
   const europeanaAggregation = edm.europeanaAggregation;
-  const providerProxy = edm.proxies.find((proxy) => {
-    return proxy.europeanaProxy === false;
-  });
+  const entities = [].concat(edm.concepts, edm.places, edm.agents).filter(checkNotNull);
```

Comment:
I recommend reducing these into an object keyed by URI so that they can be looked up thus by `lookupEntities` making `matchEntity` redundant and eliminating its repeated scanning through all entities:

```js
const entities = [].concat(edm.concepts, edm.places, edm.agents)
  .filter(checkNotNull)
  .reduce((memo, entity) => {
    memo[entity.about] = entity;
    return memo;
  }, {});
```